buildSrc/src/main/kotlin/com/toasttab/protokt/gradle/ProtobufBuild.kt

```diff
@@ -91,8 +92,9 @@ private fun extraClasspath(project: Project, task: GenerateProtoTask): String {
 private fun configureSources(project: Project, generatedSourcesPath: String) {
     val protoktDir = "$generatedSourcesPath/main/protokt"
 
-    project.the<JavaPluginConvention>()
-        .sourceSets.main.kotlin.srcDirs
+    project.the<SourceSetContainer>()["main"]
+        .java
```

Comment:
I'm not sure if the Java source set is good enough here or if it really does have to be the Kotlin source set. I can't figure out a non-deprecated way of accessing the Kotlin source set.